src/main/java/com/epam/service/HotelService.java

```diff
@@ -0,0 +1,37 @@
+package com.epam.service;
+
+import com.epam.model.Hotel;
+import com.epam.repository.HotelDAO;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+
+
+@Service
+public class HotelService {
+
+    private final HotelDAO hotelDAO;
+    @Autowired
+    public HotelService(HotelDAO hotelDAO) {
+        this.hotelDAO = hotelDAO;
+    }
+
+    public List<Hotel> getHotels(){ return hotelDAO.getHotels(); }
+
+    public List<Hotel> getHotelsByCountry (String country) { return hotelDAO.getHotelsByCountry(country); }
+
+    public void createHotel(Hotel hotel) {
```

Comment:
Please, add the validation of the args and returned results. Also will be great to add logging here

Reply:
If u add validation then would be good add it and in the update method